controllers/articles.controller.js

```diff
@@ -4,11 +4,15 @@ const { fetchArticleById, fetchArticles } = require("../models/article.models");
 const getArticleById = (req, res, next) => {
   const { article_id } = req.params;
 
+  if (isNaN(parseInt(article_id, 10))) {
+    return res.status(400).send({ msg: "Invalid article ID type" });
+  }
```

Comment:
YOu don't need to do this here, PSQL will give you an error if you give it an invalid ID Type